src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api/Requests/DataSetQueryLocation.cs

```diff
@@ -75,7 +75,7 @@ public static DataSetQueryLocation Parse(string location)
                 nameof(DataSetQueryLocationSchool.Urn) =>
                     new DataSetQueryLocationSchool { Urn = value },
 
-                nameof(DataSetQueryLocationSchool) =>
+                nameof(DataSetQueryLocationSchool.LaEstab) =>
```

Comment:
Haha, how did we get away with this one for so long?

Reply:
This is public API query endpoint work that is currently in progress and has already been fixed on my branch :wink: